doc/news/_2016-12-17_website_release.md

```diff
@@ -0,0 +1,118 @@
+# Website Release #
+
+- guid: 102b84a3-c41e-485c-8fe2-f12a24b3fbfd
+- author: Marvin Mall
+- pubDate: <TODO>
+- shortDesc: introduces new Elektra website & snippet sharing
+
+## Introduction ##
+
+With Elektra developing into a more and more reliable as well as
+popular system to manage system configurations, the demand for a
+better public appearance increases as well. For this reason, we
+are happy to be able to announce the release of our new website!
+
+The new website does not only give us a chance to better present
+ourselves to the open world, it also enables us to structure our
+project documentation better. We hope that this will make it easier
+for our users to get started with Elektra and all of its awesome
+features!
+
+Besides the documentation, the website does also include a database
+that can be used to share, search, download and convert configuration
+snippets in various formats. We hope that this tool helps developers
+and administrators, but also normal users to simplify their
+configuration processes when they have to look for a specific
+configuration snippet.
```

Comment:
Add more reasons why someone should *share* their snippets.

Some possible reasons would be if they can

1. easily search their snippets (full text?)
2. easily get it via curl and curlget resolver or
3. if its available in a git repo.